ceph.spec.in

```diff
@@ -676,7 +676,7 @@ BuildArch:	noarch
 Group:          System/Filesystems
 %endif
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
-Requires:       python%{python3_pkgversion}-remoto
+Requires:       python%{python3_pkgversion}-asyncssh
```

Comment:
ceph-mgr-cephadm is listed at https://github.com/ceph/ceph/blob/d686621fe78b47bdb0a6ca602824c5b823951d36/qa/packages/packages.yaml#L48

so, by default, this package is installed when performing teuthology tests.

```
2021-07-25T03:07:04.270 INFO:teuthology.orchestra.run.smithi188.stderr:Error:
2021-07-25T03:07:04.270 INFO:teuthology.orchestra.run.smithi188.stderr: Problem: conflicting requests
2021-07-25T03:07:04.270 INFO:teuthology.orchestra.run.smithi188.stderr:  - nothing provides python3-asyncssh needed by ceph-mgr-cephadm-2:17.0.0-6381.gf7cd730b.el8.noarch
2021-07-25T03:07:04.271 INFO:teuthology.orchestra.run.smithi188.stdout:(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
2021-07-25T03:07:04.342 DEBUG:teuthology.orchestra.run:got remote process result: 1
```

Reply:
@melissa-kun-li you might want to work with @djgalloway to add a change like https://github.com/ceph/ceph-build/pull/1454 to install the copr repo to the test slave nodes, so that we can install the dependency for non-containerized tests.  i think the new place to add the repo is https://github.com/ceph/ceph-build/blob/master/ansible/examples/slave.yml#L436 .

Reply:
thanks for the help @tchaikov !

Reply:
ceph/ceph-build/pull/1885 was merged, so I think should be ready to test again